packages/components/src/internal/components/buttons/AddEntityButton.tsx

```diff
@@ -1,32 +1,24 @@
-/*
- * Copyright (c) 2019 LabKey Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
 import React, { FC, memo } from 'react';
 
 import { ActionButton, ActionButtonProps } from './ActionButton';
 
 export interface AddEntityButtonProps extends ActionButtonProps {
+    asButton?: boolean;
     entity: string;
 }
 
-export const AddEntityButton: FC<AddEntityButtonProps> = memo(({ entity, ...actionButtonProps }) => {
-    return (
-        <ActionButton {...actionButtonProps}>
+export const AddEntityButton: FC<AddEntityButtonProps> = memo(({ entity, asButton = true, ...actionButtonProps }) => {
```

Comment:
The naming here is going to bug me when you can create a button that's not a button, but I don't have a great alternative suggestion (`AddEntityElement`? Not great.).

Reply:
yeah, I don't love this either. It isn't much code to share/duplicate so I am just going to create a new component for this so we will have `AddEntityButton` and `AddEntityElement`